Changelog.md

```diff
@@ -1,7 +1,7 @@
 ### 0.6.3 (unreleased)
 
 Language Features:
-
+ * AST: Add a new node for doxygen-style, structured documentation that can be received by contract, function, event and modifier definitions.
```

Comment:
I would say this is a compiler feature.

Reply:
Adding it to VariableDeclaration is a next step?

Reply:
Makes sense.

Reply:
> Adding it to VariableDeclaration is a next step?

I would say yes, because that's a low hanging fruit. The step after would be to properly report source locations (code is there, PR will follow)